contribute/backend/recommended-practices.md

```diff
@@ -1,169 +1,151 @@
 
-- [Cfg struct](https://github.com/grafana/grafana/blob/df917663e6f358a076ed3daa9b199412e95c11f4/pkg/setting/setting.go#L210)
-- [Injection example](https://github.com/grafana/grafana/blob/c9773e55b234b7637ea97b671161cd856a1d3d69/pkg/services/cleanup/cleanup.go#L34)
+- [`Cfg` struct](https://github.com/grafana/grafana/blob/df917663e6f358a076ed3daa9b199412e95c11f4/pkg/setting/setting.go#L210)
+- [Injection](https://github.com/grafana/grafana/blob/c9773e55b234b7637ea97b671161cd856a1d3d69/pkg/services/cleanup/cleanup.go#L34)
 
-## 103 - Reduce the use of GoConvey
+## 103 - Reduce use of GoConvey
 
 **State:** Completed.
 
-We want to migrate away from using GoConvey. Instead, we want to use
-stdlib testing with [testify](https://github.com/stretchr/testify),
-because it's the most common approach in the Go community, and we think
-it will be easier for new contributors. Read more about how we want to
-write tests in the [style guide](/contribute/backend/style-guide.md).
+We encourage you to migrate away from using GoConvey.
+Instead, we suggest the use of `stdlib` testing with [testify](https://github.com/stretchr/testify), because it's the most common approach in the Go community, and we think it will be easier for new contributors.
+To learn more about how we want to write tests, refer to the [backend style guide](/contribute/backend/style-guide.md).
 
 ## 104 - Refactor SqlStore
 
 **State:** Completed.
 
-The `sqlstore` handlers all use a global xorm engine variable. Refactor them to use the `SqlStore` instance.
+The `sqlstore` handlers all use a global `xorm` engine variable. Refactor them to use the `SqlStore` instance.
 
 ## 105 - Avoid global HTTP handler functions
 
 **State:** Ongoing, passive.
 
-Refactor HTTP handlers so that the handler methods are on the HttpServer instance or a more detailed handler struct. E.g (AuthHandler). This ensures they get access to HttpServer service dependencies (and Cfg object) and can avoid global state.
+Refactor HTTP handlers so that the handler methods are on the `HttpServer` instance or a more detailed handler struct. For example, `AuthHandler`.
+Doing so ensures that the handler methods get access to `HttpServer` service dependencies (and its `Cfg` object), so that global state may be avoided.
 
-## 106 - Date comparison
+## 106 - Compare dates
 
 **State:** Ongoing, passive.
 
-Store newly introduced date columns in the database as epoch based
-integers (i.e. unix timestamps) if they require date comparison. This
-permits a unified approach for comparing dates against all the supported
-databases instead of handling dates differently for each database. Also,
-by comparing epoch based integers, we no longer need error pruning
-transformations to and from other time zones.
+Store newly introduced date columns in the database as epoch-based integers (that is, Unix timestamps) if they require date comparison.
```

Comment:
Here, Google style recommends using "Unix epoch timestamps"

Reply:
existing style is "Unix", no change needed

Reply:
I just re-read and although it specifies Unix epoch time, I'm not 100% clear that means we should also use Unix epoch timestamps or if Unix timestamp is fine.

https://developers.google.com/style/word-list#unix-epoch-time